src/commands/Info.ts

```diff
@@ -19,7 +19,9 @@ export default class Info extends Command
 
 	public action(message: Message, args: Array<string | number>, mentions: User[], original: string): any
 	{
-		message.delete();
+		const prefix: string = !dm ? message.guild.storage.getSetting('prefix') : '';
```

Comment:
Oh, also, `dm` is undefined. Who's the tired one now? `message.guild` will suffice as a check.